x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -153,26 +173,14 @@ const HostsTableComponent = React.memo<HostsTableProps>(
         itemsPerRow={rowItems}
         limit={limit}
         loading={loading}
-        loadPage={newActivePage => loadPage(newActivePage)}
+        loadPage={loadPage}
```

Comment:
Just want to double check, but this was just an unnecessary wrapping that had been proliferated across the codebase, correct? As it gets wrapped at the query level:

https://github.com/elastic/kibana/blob/19867b9709031f3fcd3af2df5387d61a3cea63a3/x-pack/legacy/plugins/siem/public/containers/query_template_paginated.tsx#L69-L71